tests/integration/test_fim/test_registry/test_registry_basic_usage/test_long_registry_path.py

```diff
@@ -51,6 +51,9 @@ def get_configuration(request):
 def extra_configuration_before_yield():
     create_registry(registry_parser[key], sub_key_1, arch)
 
+def extra_configuration_after_yield():
+    delete_registry(registry_parser[key], sub_key_1, arch)
```

Comment:
I think instead of using these 2 functions we can simply do:
```py
test_regs = [os.path.join(key, sub_key_1)}
```

Registry paths added to `test_regs` get automatically create here:
https://github.com/wazuh/wazuh-qa/blob/ef1723d37835705eeceeb1a7f2abc0c1ef4ceec1/tests/integration/conftest.py#L453-L459

And removed here:
https://github.com/wazuh/wazuh-qa/blob/ef1723d37835705eeceeb1a7f2abc0c1ef4ceec1/tests/integration/conftest.py#L490-L494

Reply:
Done in 09002ad